### ISSUE TITLE
Getting started update docker minimal (Documentation)

### ISSUE THREAD

**Issue by naxty** [facebookarchive/caffe2#1331]
I just went through the getting started guide on the caffe2 page. 

When I came to the docker section for minimal image, I run as written in my terminal the following command.
`docker pull caffe2ai/caffe2:cpu-minimal-ubuntu14.04`

The tag cpu-minimal-ubuntu14.04 does not exist in the docker repository.

**Comment by drpalaric** [comment]
@naxty I believe you can get started using one of the minimal versions listed in the tags section of their Docker repo. The tutorial screwed up for me too by the way, which is why I looked at the Docker repo instead. Here are the images that should work for you:

Ubuntu 16.04
`docker pull caffe2ai/caffe2:c2v0.8.1.cpu.min.ubuntu16.04`

Ubuntu 14.04
`docker pull caffe2ai/caffe2:c2v0.8.1.cpu.min.ubuntu14.04`

Then you can run the tests.

**Comment by naxty** [comment]
Yes, after I was screwed up with the tutorial. I looked at the docker repo. Just want to let the maintainer know, that the tutorial is not up to date, in order to prevent beginners who dont have the knowledge to do the same mistakes.

Anyway, thanks for your reply
